catalog/users/fujitsu.json

```diff
@@ -0,0 +1,7 @@
+{
+  "id": "fujitsu",
+  "kind": "solutionprovider",
+  "name": "Fujitsu Limited",
+  "email": "fj-TandT-inquiry@dl.jp.fujitsu.com",
+  "website": "https://www.fujitsu.com"
+}
```

Comment:
@ta-Kuwayama - Thanks for submitting this PR. I see that the logo attribute is not mentioned in the user json. Please see an existing example [here](https://github.com/wbcsd/pact-catalog/blob/main/catalog/users/nri.json)

Reply:
@prasadt1 
Is logo a required option? README.md says optional.
We cannot prepare a publicly available corporate logo url
Is it possible to write "logo": "if it is not ready?


4. Fill in your user's details
...
  "logo": "<url-to-your-logo>" // to be displayed in the collaborators page (optional)

Reply:
@ta-Kuwayama 
Ideally if you have a logo url then it is recommended, since our code will fetch the Logo from there. If it is not available for fujistsu, we can try without this attribute. No changes needed then. Thanks!